Composer/packages/client/src/recoilModel/dispatchers/provisionQNA.ts

```diff
@@ -6,7 +6,7 @@ import { CallbackInterface, useRecoilCallback } from 'recoil';
 import { CognitiveServicesManagementClient } from '@azure/arm-cognitiveservices';
 import { TokenCredentials } from '@azure/ms-rest-js';
 import { SearchManagementClient } from '@azure/arm-search';
-import { SkuName } from '@azure/arm-search/lib/models';
+import { SkuName } from '@azure/arm-search/src/models';
```

Comment:
This doesn't seem right. What is the reasoning behind this change? Reminds of importing a Fluent component from `@fluent/src/Button` vs `@fluent/lib/Button` in which the latter is favorable.

Reply:
Right.
Seems this package has newer version in yarn-berry which doesn't have lib folder. Changed to pull model type from esm folder instead of src.